weblate_web/templates/features.html

```diff
@@ -2,8 +2,8 @@
 {% load i18n %}
 {% block title %}{% trans "Weblate features" %}{% endblock %}
 {% block header %}{% trans "Weblate features" %}{% endblock %}
-{% block description %}{% blocktrans %}Weblate is a feature rich computer aided translation tool. It saves time of both, developers and translators.{% endblocktrans %}{% endblock %}
-{% block headline %}{% blocktrans %}Weblate is a feature rich computer aided translation tool. It saves time of both, developers and translators. View <a href="https://docs.weblate.org/">complete documentation</a> to know more, and make your users happier!{% endblocktrans %}{% endblock %}
+{% block description %}{% blocktrans %}As a feature rich computer aided translation tool, Weblate saves both developers and translators time.{% endblocktrans %}{% endblock %}
+{% block headline %}{% blocktrans %}As a feature rich computer aided translation tool, Weblate saves both developers and translators time. View the <a href="https://docs.weblate.org/">complete documentation</a> to learn more, and make your users happier!{% endblocktrans %}{% endblock %}
```

Comment:
```suggestion
{% block headline %}{% blocktrans %}As a feature rich computer aided translation tool, Weblate saves both developers and translators time. Make your users happier!{% endblocktrans %}{% endblock %}
```
This might be used elsewhere, but the "show documentation" part is already listed below this string on https://weblate.org/nb/features/

Reply:
The separate button will be removed in the new website version, so it should stay in the text.